src/MudBlazor/Components/Button/MudFabMenu.razor.cs

```diff
@@ -0,0 +1,212 @@
+
+    /// <summary>
+    /// Occurs when <see cref="Open"/> has changed.
+    /// </summary>
+    [Parameter, Category(CategoryTypes.Menu.PopupBehavior)]
+    public EventCallback<bool> OpenChanged { get; set; }
+
+    /// <summary>
+    /// Sets the menu to a fixed position in the bottom right corner of the screen with padding of 16 px towards each screen edge.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <c>false</c>.
+    /// </remarks>
+    [Parameter, Category(CategoryTypes.Button.Behavior)]
+    public bool Fixed { get; set; }
+
+    /// <summary>
+    /// Whether to replace the set icon with a close icon when the menu is open.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <c>true</c>.
+    /// </remarks>
+    [Parameter, Category(CategoryTypes.Button.Behavior)]
+    public bool UseCloseIcon { get; set; } = true;
+
+    /// <summary>
+    /// Dampens the background color of the menu items when set to true.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <c>true</c>.
+    /// </remarks>
+    [Parameter, Category(CategoryTypes.Button.Behavior)]
+    public bool DampenItemColors { get; set; } = true;
+
+    /// <summary>
+    /// The alignment of the menu items in respect to the menu button.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <c>AlignItems.Center</c>.
+    /// </remarks>
+    [Parameter, Category(CategoryTypes.Button.Behavior)]
+    public AlignItems AlignItems { get; set; } = AlignItems.Center;
+
+    /// <summary>
+    /// Whether to open the menu on mouse hover.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <c>true</c>.
+    /// </remarks>
+    [Parameter, Category(CategoryTypes.Menu.Behavior)]
+    public bool OpenOnMouseHover { get; set; } = true;
+
+    /// <summary>
+    /// Whether to close the menu when a menu item is clicked.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <c>true</c>.
+    /// </remarks>
+    [Parameter, Category(CategoryTypes.Menu.Behavior)]
+    public bool CloseOnMenuItemClicked { get; set; } = true;
+
+    public MudFabMenu()
+    {
+        using var registerScope = CreateRegisterScope();
+        _openState = registerScope.RegisterParameter<bool>(nameof(Open))
+            .WithParameter(() => Open)
+            .WithEventCallback(() => OpenChanged)
+            .WithChangeHandler(HandleOpenChanged);
+    }
+
+    protected override void OnParametersSet()
+    {
+        base.OnParametersSet();
+
+        if (!_openState.Value || !UseCloseIcon)
+        {
+            _startIcon = StartIcon;
+            _endIcon = EndIcon;
+        }
+    }
+
+    private void HandleOpenChanged(ParameterChangedEventArgs<bool> args) => HandleOpenChanged(args.Value);
+
+    private void HandleOpenChanged(bool open)
+    {
+        if (open && UseCloseIcon)
+        {
+            if (StartIcon != null) _startIcon = Icons.Material.Outlined.Add;
+            if (EndIcon != null) _endIcon = Icons.Material.Outlined.Add;
```

Comment:
On its own line surrounded with braces

```suggestion
            if (StartIcon != null)
            {
                _startIcon = Icons.Material.Outlined.Add;
            }

            if (EndIcon != null)
            {
                _endIcon = Icons.Material.Outlined.Add;
            }
```